observe_configure_script.sh

```diff
@@ -538,6 +562,27 @@ if [ "$appgroup" != UNSET ]; then
     sed -i "s/#REPLACE_WITH_OBSERVE_APP_GROUP_OPTION/Record appgroup ${appgroup}/g" ./*
 fi
 
+metadata_buffer_size="8mb"
+metadata_interval_secs="300"
+sed -i "s/REPLACE_WITH_METADATA_BUFFER_SIZE/${metadata_buffer_size}/g" ./*
+sed -i "s/REPLACE_WITH_METADATA_INTERVAL/${metadata_interval_secs}/g" ./*
+
+metadata_command=":"
+if [[ "$cloud_metadata" == TRUE ]]; then
+  if [ $datacenter == "AWS" ]; then
```

Comment:
Is there any way to auto detect the cloud we are on?  I worry about fragility of requiring the correct datacenter flag to make this work.

Reply:
thanks, i think the easiest way is to run the curl command with a timeout and then check the bash status and http status. i'll try that out